$NetBSD: patch-Makefile,v 1.4 2026/10/27 13:55:03 adam Exp $

Let pkgsrc choose optimisation flags.
Fix the man page path.

--- Makefile.orig	2026-10-25 08:02:02.000000000 +0000
+++ Makefile
@@ -234,7 +234,7 @@ override GOODFLAGS := $(foreach flag,$(T
 #? Flags, Libraries and Includes
 override REQFLAGS   := -std=c++23
 WARNFLAGS			:= -Wall -Wextra -pedantic
-OPTFLAGS			:= -O2 $(LTO)
+OPTFLAGS			:= $(LTO)
 LDCXXFLAGS			:= -pthread -DFMT_HEADER_ONLY -D_GLIBCXX_ASSERTIONS -D_LIBCPP_HARDENING_MODE=_LIBCPP_HARDENING_MODE_DEBUG -D_FILE_OFFSET_BITS=64 $(GOODFLAGS) $(ADDFLAGS)
 override CXXFLAGS	+= $(REQFLAGS) $(LDCXXFLAGS) $(OPTFLAGS) $(WARNFLAGS)
 override LDFLAGS	+= $(LDCXXFLAGS) $(OPTFLAGS) $(WARNFLAGS)
@@ -381,9 +381,9 @@ install:
 	@mkdir -p $(DESTDIR)$(PREFIX)/share/icons/hicolor/scalable/apps
 	@cp -p Img/icon.svg $(DESTDIR)$(PREFIX)/share/icons/hicolor/scalable/apps/btop.svg
 ifneq ($(wildcard btop.1),)
-	@$(call green,Installing man page to: ,$(WHITE)$(DESTDIR)$(PREFIX)/share/man/man1/btop.1)
-	@mkdir -p $(DESTDIR)$(PREFIX)/share/man/man1
-	@cp -p btop.1 $(DESTDIR)$(PREFIX)/share/man/man1/btop.1
+	@$(call green,Installing man page to: ,$(WHITE)$(DESTDIR)$(PREFIX)/$(PKGMANDIR)/man1/btop.1)
+	@mkdir -p $(DESTDIR)$(PREFIX)/$(PKGMANDIR)/man1
+	@cp -p btop.1 $(DESTDIR)$(PREFIX)/$(PKGMANDIR)/man1/btop.1
 endif
 
 #? Set SUID bit for btop as $SU_USER in $SU_GROUP
@@ -412,8 +412,8 @@ uninstall:
 	@rm -rfv $(DESTDIR)$(PREFIX)/share/icons/hicolor/48x48/apps/btop.png
 	@$(call red,Removing: ,$(WHITE)$(DESTDIR)$(PREFIX)/share/icons/hicolor/scalable/apps/btop.svg)
 	@rm -rfv $(DESTDIR)$(PREFIX)/share/icons/hicolor/scalable/apps/btop.svg
-	@$(call red,Removing: ,$(WHITE)$(DESTDIR)$(PREFIX)/share/man/man1/btop.1)
-	@rm -rfv $(DESTDIR)$(PREFIX)/share/man/man1/btop.1
+	@$(call red,Removing: ,$(WHITE)$(DESTDIR)$(PREFIX)/$(PKGMANDIR)/man1/btop.1)
+	@rm -rfv $(DESTDIR)$(PREFIX)/$(PKGMANDIR)/man1/btop.1
 
 #? Pull in dependency info for *existing* .o files
 -include $(OBJECTS:.$(OBJEXT)=.$(DEPEXT))
